translate/storage/ts2.py

```diff
@@ -349,7 +350,7 @@ def merge(self, otherunit, overwrite=False, comments=True, authoritative=False):
             self.markfuzzy(False)
 
     def isobsolete(self):
-        return self._gettype() == "obsolete"
+        return self._gettype() == "obsolete" or self._gettype() == "vanished"
```

Comment:
Can you please add test covering this? Apparently this method is not tested at all.

Reply:
Do you mean adding the test case into existing unit test to verify this feature? Ok, I'll add that once I'll reach my PC.